cmd/maya-apiserver/spc-watcher/storagepool_delete.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/openebs/maya/pkg/storagepool"
 )
 
+
+// DeleteStoragePool receives StoragePoolClaim object and deletes it. 
```

Comment:
Please put it in following way :
`// DeleteStoragePool receives StoragePoolClaim delete event and calls the required handlers
to delete other pool related resources.`

Reply:
Sure, addressed it in #667. PTAL.